conduction/commonground-bundle/config/conduction_common_ground.yaml

```diff
@@ -0,0 +1,35 @@
+conduction_common_ground:      
+    bag:
+        # cashing is a common ground hot patato, from a data at the source perspective we don't want to duplicate data
```

Comment:
I don't know anything about the bundle, yet I'm not sure this comment helps me in any way, sorry :)
Any more factual alternatives (or is the bundle about patatoes? )